src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -0,0 +1,36 @@
+import './radio-button-group.scss';
+import React from 'react';
+import { Button } from '../../components/button/button';
+import { DropDownOption } from '../drop-down-options/drop-down-options';
+
+// Todo: they are the same, maybe generalize??
```

Comment:
hm yeah, key-value pairs are super generic, i know java/kotlin has a `Pair` class that does exactly that -- and id,value pair

Reply:
haha, I'm an idiot. We can just use Record<string, React.ReactNode> which is exactly that. I don't know why I didn't think about that eariler